src/main/scala/microsites/MicrositeKeys.scala

```diff
@@ -167,14 +201,64 @@ trait MicrositeAutoImportSettings extends MicrositeKeys {
 
   lazy val fr = new FileReader
 
+  def createVersionsJson(targetDir: String, content: List[Version]): File = {
+    val jekyllDir  = "jekyll"
+    val targetPath = s"$targetDir$jekyllDir/_data/versions.json"
+    createFile(targetPath)
+    writeContentToFile(content.asJson.toString, targetPath)
```

Comment:
You might want to write the json with a circe Printer:

```scala
content.asJson.pretty(Printer.noSpaces) // Or
content.asJson.pretty(Printer.spaces2)
```

Reply:
Just curious, what is the benefit?  This is the format being printed now:

```json
[
  {
    "name" : "jv-116-multiversion",
    "own" : true
  },
  {
    "name" : "v0.9.7",
    "own" : false
  }
]
```

I guess `asJson` is already using `spaces2` internally?

https://github.com/circe/circe/blob/master/modules/core/shared/src/main/scala/io/circe/Json.scala#L188

Reply:
Yup, it seems like :) . Thanks for checking.